staging/src/k8s.io/kube-scheduler/config/v1alpha2/types_pluginargs.go

```diff
@@ -173,3 +173,13 @@ type ServiceAffinityArgs struct {
 	// +listType=atomic
 	AntiAffinityLabelsPreference []string `json:"antiAffinityLabelsPreference,omitempty"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// VolumeBindingArgs holds arguments used to configure the VolumeBinding plugin.
+type VolumeBindingArgs struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// BindTimeoutSeconds is the timeout in seconds in volume binding.
```

Comment:
document the default

Reply:
does the same doc as BindTimeoutSeconds apply?

> Duration to wait for a binding operation to complete before timing out.
> Value must be non-negative integer. The value zero indicates no waiting.

Reply:
no accurate, it's better to be clear that 

- the unit is `second`
- use `volume` before `binding` to distinguish it from pod `binding` operation in the scheduler

Reply:
docs about the requirement, special value (`0`) and the default have been updated